pkg/operator/ceph/test/containers.go

```diff
@@ -77,6 +77,13 @@ func (ct *ContainersTester) AssertArgsContainCephRequirements() {
 	}
 }
 
+// RequireAdditionalEnvVars adds a list of environment variable names to the list of required
+// variables for a single unit test (it does not persist between different tests).
+// Usage: myPodTemplateSpecTester.Spec().Containers().RequireAdditionalEnvVars("I_AM", "REQUIRED")
+func (*ContainersTester) RequireAdditionalEnvVars(varNames ...string) {
+	requiredEnvVars = append(requiredEnvVars, varNames...)
```

Comment:
@BlaineEXE  When the requiredEnvVars are modified, it makes the test fail for other two init containers that are there in the podTemplate. We need the additional variables only when the container name is "mgr", but if we update the env vars for init containers as well then the test will pass.

Reply:
I don't see a particular issue adding the vars to the init containers as well. Though I could understand wanting to not have them there. Do you have an opinion @travisn ?

Reply:
Adding them to the init container(s) sounds ok even if they are ignored there